### ISSUE TITLE
More pronounced visual difference between == and ===

### ISSUE THREAD

**Issue by schlessera** [i-tu/Hasklig#63]
In some languages (PHP, JavaScript), there's a big difference between the two following comparison operators:

`==` : compare if same value
`===` : compare if same value and same type

This makes it important to easily recognize which one you are using.

The current ligatures make the `===` operator a slightly longer version of the `==`, and it is difficult to distinguish them at first glance.

In some other ligature fonts, the `===` is represented as three horizontal bars, instead of only two. This makes it immediately distinguishable from the standard two bars in `==`.

The negated versions of these, `!=` and `!==`, should get the same treatment.

Here's an example of how this looks in the Firacode font (https://github.com/tonsky/FiraCode):
![screenshot 2016-07-13 15 33 24](https://cloud.githubusercontent.com/assets/83631/16805206/633f0996-490f-11e6-8f8d-7fc8182fc83d.png)

**Comment by smbkr** [comment]
I have to agree with this - the purpose of the ligatures is to make the text more readable at a glance.

Monoid (https://github.com/larsenwork/monoid) takes the opposite approach to Fira Code, where the equals operators are given more spacing, to make the difference between == and === easily spotted.
![1-y2dwpffgj7jtnhjaeto6zg](https://cloud.githubusercontent.com/assets/933535/18987195/71724768-86f8-11e6-92a6-fdc189def4ad.png)

**Comment by mistadikay** [comment]
Any updates on this?

**Comment by batjko** [comment]
I'm all for Fira Code's convention here. It's both very obvious (compared to just a somewhat elongated normal equality operator) and also very pleasing to the eye. It should be standard for all these ligature code fonts.

**Comment by gnhuy91** [comment]
+1 would love to have this, the only thing i want from Fira Code.

**Comment by jonnywilliamson** [comment]
Yes, the == and === are very similar and having a 3rd bar on === makes it so much clearer!

This would be a very much appreciated improvement.

**Comment by killua99** [comment]
+1 Kinda hard to know when I'm using === or ==, it does need more difference

**Comment by batjko** [comment]
Seems like this repo / font isn't being maintained much anymore.
Has anybody found alternative fonts that have this, apart from Fira Code?
